include/msgpack/v1/object.hpp

```diff
@@ -466,12 +466,13 @@ struct object_stringize_visitor {
         m_os << '"';
         return true;
     }
-    bool visit_bin(const char* v, uint32_t size) {
-        (m_os << '"').write(v, static_cast<std::streamsize>(size)) << '"';
+    bool visit_bin(const char* /*v*/, uint32_t size) {
+        m_os << "\"BIN(size:" << size << ")\"";
         return true;
     }
-    bool visit_ext(const char* /*v*/, uint32_t /*size*/) {
-        m_os << "EXT";
+    bool visit_ext(const char* v, uint32_t size) {
+        int type = ((size > 0) && (v[0] >= 0)) ? v[0] : -1;
```

Comment:
Maybe my comment is misleading. If the value of type is `-2` the output should be `-2`. I mean print as **singed**.